docs/source/index.rst

```diff
@@ -17,6 +17,7 @@ Table of contents
        gpkit101
        installation
        gettingstarted
+       debugging

```

Comment:
I propose `debugging` be moved after `advancedcommands` (and perhaps even after `signomialprogramming`)